site_root/input/resources/profiles/StructureDefinition-kfdrc-phenotype.json

```diff
@@ -0,0 +1,48 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-phenotype",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-phenotype",
+  "version": "0.1.0",
+  "name": "kfdrc-phenotype",
+  "title": "Kids First DRC Phenotype",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Observation",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Observation",
+        "path": "Observation"
+      },
+      {
+        "id": "Observation.extension",
+        "path": "Observation.extension"
+      },
+      {
+        "id": "Observation.extension:age-at-event",
+        "path": "Observation.extension",
+        "sliceName": "age-at-event",
+        "sliceIsConstraining": false,
+        "label": "age_at_event",
+        "short": "Age time of event",
+        "definition": "Age (in days since birth) of the Participant at time of event.",
```

Comment:
Similar comment to the one made on `kfdrc-specimen.json`. This should probably be more specific. This `age-at-event` refers to time of observation.

Reply:
Updated!